AmplifyPlugins/Predictions/CoreMLPredictionsPlugin/CoreMLPredictionsPlugin+ClientBehavior.swift

```diff
@@ -90,14 +90,18 @@ extension CoreMLPredictionsPlugin {
             )
             let stream = AsyncThrowingStream<Predictions.Convert.SpeechToText.Result, Error> { continuation in
                 Task {
-                    let result = try await coreMLSpeech.getTranscription(
```

Comment:
The error thrown here was not being captured. However, there may be more than one way to propagate these errors.